crates/egui-winit/src/lib.rs

```diff
@@ -817,19 +819,24 @@ impl State {
         }
 
         if let Some(ime) = ime {
-            let rect = ime.rect;
             let pixels_per_point = pixels_per_point(&self.egui_ctx, window);
-            crate::profile_scope!("set_ime_cursor_area");
-            window.set_ime_cursor_area(
-                winit::dpi::PhysicalPosition {
-                    x: pixels_per_point * rect.min.x,
-                    y: pixels_per_point * rect.min.y,
-                },
-                winit::dpi::PhysicalSize {
-                    width: pixels_per_point * rect.width(),
-                    height: pixels_per_point * rect.height(),
-                },
-            );
+            let rect = ime.rect * pixels_per_point;
```

Comment:
a `Rect` is generally assumed to use `points` as the unit. It doesn't matter much here _rigth now_, but if someone were to try to use `self.ime_rect` for something else (e.g. debug-paint it) they would be in for weird bugs.

So I suggest postposning the multiplication with `pixels_per_point` until the conversion to `PhysicalPositon/PhysicalSize`, just like in the old code

Reply:
Alright, but I changed it to store the `pixels_per_point` as well in case the user changes their screen resolution in-between two times the app renders

Reply:
Ah I see, that's what you wanted to capture… then I think the old code was better, if we call it `ime_rect_in_pixels` or similar. This tuple business is a bit over complex